DiamondLightSource/python-dodal
# 2: Add Artemis devices to dodal

Fixes #2

To test:
* Confirm unit tests pass (by running `pytest tests/ -m "not s03"`)
* Confirm structure of i03/s03 looks correct
* Confirm history is somewhat preserved

Note that this adds devices as is from Artemis, there may well be improvements that could be made to this code but this PR should focus on getting the code into `dodal`, subsequent issues can be made to tidy it up.
